static/app/components/charts/utils.tsx

```diff
@@ -185,10 +185,9 @@ export function shouldFetchPreviousPeriod({
  * Generates a series selection based on the query parameters defined by the location.
  */
 export function getSeriesSelection(
-  location: Location,
-  parameter = 'unselectedSeries'
```

Comment:
since it's always going to be `unselectedSeries`, the parameter is no longer needed to be dynamic